Practice/kruglov/home_task_4/PyLec4T6.py

```diff
@@ -0,0 +1,31 @@
+# 6 PyLecT6.py
+
+def del_find(el, mtx):
+    i = 0
+    x = []
+    for row in mtx: 
+        while i < len(row):
+            if el == row[i] and i not in x:
+                x.append(i)
+            i += 1
+        i = 0
+    x.sort()
+    for row1 in mtx:
+        cv = len(row)
+        while i < cv:
+            if row1[i] not in x:
+                print(row1[i])
```

Comment:
It would be better if del_find function returned updated matrix rather than just printed the result.

Reply:
I agree, but I could do it before the prompt)